### PR TITLE
rbdmap: add manpage

### PR DESCRIPTION
**Pull request by smithfarm**
http://tracker.ceph.com/issues/15212 Fixes: #15212

Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

File: doc/man/8/rbdmap.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ rbdmap -- map RBD devices at boot time
+=========================================
+
+.. program:: rbdmap
+
+Synopsis
+========
+
+| **rbdmap**
+
```
Comment (branto1) [inline doc/man/8/rbdmap.rst]: We could use the basic usage info, here. I.e. rbdmap map | unmap.

File: doc/man/8/rbdmap.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Description
+===========
+
+**rbdmap** is a shell script that is intended to be run at boot time
+by the init system (sysvinit, upstart, systemd). It looks for an
```
Comment (branto1) [inline doc/man/8/rbdmap.rst]: While it is true that it is usually run by init system, a user can also easily run it to achieve the same thing without having to have it run every time a computer starts -- that is why I originally kept it as a script in /usr/bin.

Alternatively, we could skip creating the man page altogether and move the script to /usr/lib/... (libexecdir). I'm not sure how useful the standalone script in bindir is and it depends mostly on that.

**Comment by smithfarm** [comment]
> While it is true that it is usually run by init system, a user can also easily run it to achieve the same thing without having to have it run every time a computer starts -- that is why I originally kept it as a script in /usr/bin.
> 
> Alternatively, we could skip creating the man page altogether and move the script to /usr/lib/... (libexecdir). I'm not sure how useful the standalone script in bindir is and it depends mostly on that.

If there is a reasonable use case for running it manually (and I think there is), we should keep it where it is. I revamped the manpage a little - can you take another look?

**Comment by branto1** [comment]
I also think there is a valid use case for this being in bindir (well, that is why I originally put it there). With the recent changes to the wording and after mentioning map/unmap in the man page, lgtm,

**Pull request merged**